client/gefyra/__main__.py

```diff
@@ -24,10 +24,16 @@
 
 up_parser = action.add_parser("up")
 up_parser.add_argument(
-    "-e",
-    "--endpoint",
-    dest="cargo_endpoint",
-    help="the Wireguard endpoint in the form <IP>:<Port> for Gefyra to connect to",
+    "--host",
```

Comment:
add short version `-h`, too? (see `psql`)

Reply:
Isn't `-h` kind of reserved for `help`?

Reply:
`-H` maybe?